Don't initialize anything other than the BSP

For unknown reasons, after 7fadaacd50d7 "UefiCpuPkg/CpuDxe: Consume
MpInitLib to produce CPU MP Protocol services" Windows 10 hangs early in
boot (in HaliHaltSystem()) when booting with > 1 vCPU on Xen.  Work
around this by not bringing up any processors other than the BSP.

---
 OvmfPkg/OvmfPkgX64.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OvmfPkg/OvmfPkgX64.dsc b/OvmfPkg/OvmfPkgX64.dsc
index bd9b569..277390d 100644
--- a/OvmfPkg/OvmfPkgX64.dsc
+++ b/OvmfPkg/OvmfPkgX64.dsc
@@ -503,7 +503,7 @@
   gEfiMdeModulePkgTokenSpaceGuid.PcdPropertiesTableEnable|FALSE
 
   # UefiCpuPkg PCDs related to initial AP bringup and general AP management.
-  gUefiCpuPkgTokenSpaceGuid.PcdCpuMaxLogicalProcessorNumber|64
+  gUefiCpuPkgTokenSpaceGuid.PcdCpuMaxLogicalProcessorNumber|1
   gUefiCpuPkgTokenSpaceGuid.PcdCpuApInitTimeOutInMicroSeconds|50000
 
 ################################################################################
-- 
2.7.4

